### PR TITLE
furmark: Add version 1.20.8.0

### PR DESCRIPTION
**Pull request by r15ch13**

### CODE REVIEW

File: bucket/furmark.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://geeks3d.com/furmark/",
+    "description": "A GPU Stress Test and OpenGL Benchmark",
```
Comment (Ash258) [inline bucket/furmark.json]:
```suggestion
    "description": "A GPU Stress Test and OpenGL Benchmark.",
```

File: bucket/furmark.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            "FurMark.exe",
+            "FurMark"
+        ]
+    ],
+    "checkver": {
+        "url": "https://geeks3d.com/furmark/downloads/",
+        "regex": "<b>Version (?<version>[\\d.]+) - (?<year>\\d+)-\\d+-\\d+</b>"
+    },
+    "autoupdate": {
+        "url": "https://www.geeks3d.com/downloads/$matchYear/FurMark_$version_Setup.exe"
```
Comment (Ash258) [inline bucket/furmark.json]: Add hash extraction from checkver url

image

Comment (r15ch13) [reply bucket/furmark.json]: Yeah, I was sure I forgot something ... 😁

File: bucket/furmark.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "checkver": {
+        "url": "https://geeks3d.com/furmark/downloads/",
+        "regex": "<b>Version (?<version>[\\d.]+) - (?<year>\\d+)-\\d+-\\d+</b>"
+    },
+    "autoupdate": {
+        "url": "https://www.geeks3d.com/downloads/$matchYear/FurMark_$version_Setup.exe",
+        "hash": {
+            "url": "https://geeks3d.com/furmark/downloads/",
+            "regex": "$basename[\\S\\s]+SHA256\\s+=>\\s+$sha256\\s+<!--"
```
Comment (Ash258) [inline bucket/furmark.json]: - In general `.*?` is prefered in other manfiests over `[\\S\\s]+`
- Is it needed to be so strict?

```suggestion
            "regex": "$basename.*?SHA256\\s+=>\\s+$sha256\\s+<!--"
```

```suggestion
            "regex": "$basename.*?+SHA256\\s+=>\\s+$sha256"
```

Comment (r15ch13) [reply bucket/furmark.json]: I can't remember it, even if it's so simple 😁

Comment (Ash258) [reply bucket/furmark.json]: You could pretty much remove the comment too. Comment could be easily changed / removed.

Comment (r15ch13) [reply bucket/furmark.json]: By using `[\S\s]+` it would skip the correct checksum and select the last one.

Comment (r15ch13) [reply bucket/furmark.json]: Hm, `.*?` doesn't work.
https://regex101.com/r/g30jIy/1

Comment (Ash258) [reply bucket/furmark.json]: You need multiline `(?sm)$basename.*?SHA256\\s+=>\\s+$sha256`

**Pull request merged**